maxentep/metapop_model.py

```diff
@@ -22,7 +22,7 @@ def _compute_trans_diagonal(tri_values, indices, shape):
     return diag
 
 
-def recip_norm_mat_dist(trans_times, trans_times_var, indices, sample_R=True):
+def recip_norm_mat_dist(trans_times, trans_times_var, indices, sample_R=True, n_infectious_compartments=1):
```

Comment:
Maybe we could call this something more informative so it's clear what effect it has? Maybe low or something?